CRM/Activity/BAO/Activity.php

```diff
@@ -1830,6 +1830,9 @@ public static function createFollowupActivity($activityId, $params) {
     $followupParams['activity_type_id'] = $params['followup_activity_type_id'];
     // Get Subject of Follow-up Activiity, CRM-4491
     $followupParams['subject'] = $params['followup_activity_subject'] ?? NULL;
+    if (!is_array($params['followup_assignee_contact_id'])) {
```

Comment:
```suggestion
    if (!is_array($params['followup_assignee_contact_id']) && !empty($params['followup_assignee_contact_id'])) {
```

otherwise when there is no assignee on the case activity followup the next line converts it to `array('')` so it then tries to insert contact id 0 and crashes.